source/use-case/todoapp/form-event/add-todo-item/src/App.js

```diff
@@ -1,20 +1,24 @@
-import { element } from "./view/html-util.js";
+import { element, render } from "./view/html-util.js";
 
 export class App {
     mount() {
         const formElement = document.querySelector("#js-form");
         const inputElement = document.querySelector("#js-form-input");
         const containerElement = document.querySelector("#js-todo-list");
         const todoItemCountElement = document.querySelector("#js-todo-count");
+        // TodoリストをまとめるList要素
+        const todoListElement = element`<ul />`;
         // Todoアイテム数
         let todoItemCount = 0;
         formElement.addEventListener("submit", (event) => {
             // 本来のsubmitイベントの動作を止める
             event.preventDefault();
             // 追加するTodoアイテムの要素(li要素)を作成する
             const todoItemElement = element`<li>${inputElement.value}</li>`;
-            // Todoアイテムをcontainerに追加する
-            containerElement.appendChild(todoItemElement);
+            // TodoアイテムをtodoListElementに追加する
+            todoListElement.appendChild(todoItemElement);
+            // containerElementの中身をtodoListElementで上書きする
```

Comment:
関数のままのコメントな気がするので、

```suggestion
            // コンテナ要素の中身をTodoリストをまとめるList要素で上書きする
```

こういうのとかはどうですかね?

Reply:
> 関数のままのコメントな気がするので、

確かにそうですね。。。

記載していただいたコメントにするよう文言を変更いたしました。
また、類似箇所がありましたのでまとめて修正しております。

https://github.com/asciidwango/js-primer/pull/1385/commits/ff3bc97fcf0a2866747223a6db68e7efa0eae6ba